libraries/AC_AttitudeControl/AC_WeatherVane.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * Aircraft Weathervane options common to vtol plane and copters
+ */
+#include <AP_HAL/AP_HAL_Boards.h>
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+#include <AP_HAL/AP_HAL.h>
+#include "AC_WeatherVane.h"
+#include <AP_Math/AP_Math.h>
+#include <GCS_MAVLink/GCS.h>
+#include <AP_Terrain/AP_Terrain.h>
+#include <AP_AHRS/AP_AHRS.h>
```

Comment:
Fairly sure you don't need all of these.

Reply:
yeah, some hangovers from old versions.  Have tidied.